lightning/src/ln/chan_utils.rs

```diff
@@ -616,12 +689,27 @@ impl Readable for LocalCommitmentTransaction {
 				_ => return Err(DecodeError::InvalidValue),
 			},
 		};
+		let local_keys = Readable::read(reader)?;
+		let feerate_per_kw = Readable::read(reader)?;
+		let htlcs_count: u64 = Readable::read(reader)?;
+		let mut per_htlc = Vec::with_capacity(cmp::min(htlcs_count as usize, MAX_ALLOC_SIZE / std::mem::size_of::<(HTLCOutputInCommitment, Option<Signature>, Option<Transaction>)>()));
```

Comment:
This doesnt compile on 1.22 - you need ::std::mem

Reply:
Should be solved at f07505e